lxd/operations/operations.go

```diff
@@ -9,15 +9,16 @@ import (
 
 	"github.com/google/uuid"
 
-	"github.com/canonical/lxd/lxd/auth"
 	"github.com/canonical/lxd/lxd/db/operationtype"
 	"github.com/canonical/lxd/lxd/events"
 	"github.com/canonical/lxd/lxd/request"
 	"github.com/canonical/lxd/lxd/response"
 	"github.com/canonical/lxd/lxd/state"
 	"github.com/canonical/lxd/shared"
 	"github.com/canonical/lxd/shared/api"
+	sharedAuth "github.com/canonical/lxd/shared/auth"
```

Comment:
auth is fine here as there's no other import also called `auth` right?

Reply:
Ah yes, there were so many references to `auth.Entitlement` in the `lxd` package I got a bit lazy and did a search/replace.